resources/helm/dask-gateway/values.schema.yaml

```
@@ -16,6 +16,16 @@ required:
   - rbac
   - global
 properties:
+  fullnameOverride:
+    type: [string, "null"]
+    description: |
+      fullnameOverride and nameOverride allow you to adjust how the resources
+      part of the Helm chart are named.
+  nameOverride:
+    type: [string, "null"]
+    description: |
+      See the documentation under [`fullnameOverride`](schema_fullnameOverride).
```

```suggestion
      See the description offullnameOverride.
```

I'm removing the technical parts that only works when rendering these descriptions in Sphinx as done in jupyterhub/zero-to-jupyterhub-k8s but not done in this project.

---

ah okay i see nice